web-console/project-details/storage-metrics.md

```diff
@@ -0,0 +1,66 @@
+# Storage usage Metrics
+
+## Open Questions
+- What should the entry for capacity consumed be called? I'm using "Used" in these mocks, but should it be "Capacity Consumed"? Something else?
+- What would we want the thresholds for the donut chart to be? Are the defaults ok?
+- Will these storage metrics be tied to any events or warnings so that we can proactively inform users of any storage problems?
+- Is `N/A` how we want to show that the metric doesn't exist? Should we add a tooltip to that as well?
+- Is the order of the fields ok for the PVC details page?
+
+## Storage List Page
+
+![storage list](img/storage-list.png)
+
+- Add a column **Used** between the **Status** and **Capacity** columns.
+- The units should match between the **Used** and **Capacity** columns for easy comparison.
+- Because it's possible that not all storage classes will have the capacity consumed metric, field-level help should be used in the column header to explain why some rows may not have an entry in that column.
+- If that metric is not available for a storage class, `N/A` should appear in the **Used** column for that row.
```

Comment:
@ncameronbritt I think in the examples from Rox she had "NA without a slash. Should we do that here to be consistent?

Reply:
NA could be construed as North America or some other abbreviation. I Think N/A is pretty standard for not applicable or not available

Reply:
+1 on N/A. I changed it for the sake of consistency with what Rox showed, N/A is what I'm accustomed to seeing.